packages/material/src/complex/MaterialEnumArrayControl.tsx

```diff
@@ -0,0 +1,115 @@
+  FormGroup,
+  FormHelperText,
+  Hidden
+} from '@material-ui/core';
+import { startCase } from 'lodash';
+import { isEmpty } from 'lodash';
+import React, { useMemo, useRef } from 'react';
+
+const toUiData = (options: any, data: any) => {
+  const result = Array(options.length);
+  options.forEach((option: any, index: number) => {
+    result[index] = data !== undefined && data.includes(option.const);
+  });
+  return result;
+};
+
+export const MaterialEnumArrayControl = ({
+  schema,
+  visible,
+  errors,
+  path,
+  data,
+  handleChange,
+  ...otherProps
+}: ControlProps) => {
+  const options = useMemo(() => (schema.items as any).anyOf, [schema]);
+  const uiData = useMemo(() => toUiData(options, data), [data, options]);
+
+  const onChange = (option: string, value: any) => {
+    const newData = data ? [...data] : [];
+    if (value) {
+      // option was added
+      newData.push(option);
+    } else {
+      // option was removed
+      const indexInData = newData.indexOf(option);
+      newData.splice(indexInData, 1);
+    }
+
+    handleChange(path, newData);
+  };
+  // we are using a ref for the onChange function because
+  // changing only the handleChange prop of MuiCheckbox does not trigger an update
+  const onChangeRef = useRef(onChange);
+  onChangeRef.current = onChange;
+  return (
+    <Hidden xlUp={!visible}>
+      <FormControl component='fieldset'>
+        <FormGroup row>
+          {uiData.map((value: any, index: number) => {
+            const option = options[index];
+            const optionPath = Paths.compose(path, `${index}`);
+            return (
+              <FormControlLabel
+                id={option.const}
+                key={option.const}
+                control={
+                  <MuiCheckbox
+                    key={'checkbox-' + option.const}
+                    isValid={isEmpty(errors)}
+                    path={optionPath}
+                    handleChange={(_childPath, newValue) =>
+                      onChangeRef.current(option.const, newValue)
+                    }
+                    data={value}
+                    errors={errors}
+                    schema={schema}
+                    visible={visible}
+                    {...otherProps}
+                  />
+                }
+                label={option.title ?? startCase(option.const)}
+              />
+            );
+          })}
+        </FormGroup>
+        <FormHelperText error hidden={isEmpty(errors)}>
+          {errors}
+        </FormHelperText>
+      </FormControl>
+    </Hidden>
+  );
+};
+
+export const materialEnumArrayControlTester: RankedTester = rankWith(
+  5,
+  and(
+    uiTypeIs('Control'),
+    and(
+      schemaMatches(
+        schema => hasType(schema, 'array') && !Array.isArray(schema.items)
+      ),
+      schemaSubPathMatches('items', schema => {
+        return schema.anyOf !== undefined && schema.anyOf.length > 0;
+      })
+    )
+  )
+);
+
+export default MaterialEnumArrayControl;
```

Comment:
To be consistent with other controls we should default export the wrapped version